cluster/options_test.go

```diff
@@ -0,0 +1,97 @@
+package cluster
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var opts = DriverOpts{"foo1=bar", "foo2=-5", "foo3=7", "foo4=0.6"}
+
+func TestString(t *testing.T) {
+	ok, val := opts.String("foo1")
+	assert.True(t, ok)
+	assert.Equal(t, val, "bar")
+
+	ok, val = opts.String("foo2")
+	assert.True(t, ok)
+	assert.Equal(t, val, "-5")
+
+	ok, val = opts.String("foo3")
+	assert.True(t, ok)
+	assert.Equal(t, val, "7")
+
+	ok, val = opts.String("foo4")
+	assert.True(t, ok)
+	assert.Equal(t, val, "0.6")
+
+	ok, val = opts.String("foo5")
```

Comment:
tiny little nit, feel free to ignore: Instead of `foo5` I'd use `invalid` or `non-existent`. It's not obvious that `foo5` doesn't exist